### ISSUE TITLE
Fix different warnings

### ISSUE THREAD

**Issue by ovalseven8** [threema-ch/threema-web#850]
## Bug Description

![Bildschirmfoto vom 2019-07-10 08-15-13](https://user-images.githubusercontent.com/8258609/60945173-28fa0900-a2eb-11e9-98cd-05c581a2cf66.png)
<details>
<summary>Warning</summary>

```
ARIA: Attribute " aria-label ", required for accessibility, is missing on node: 
button.md-icon-button.md-button.md-ink-ripple
​
accessKey: ""
​
accessKeyLabel: ""
​
assignedSlot: null
​
attributes: NamedNodeMap(10) [ class="md-icon-button md-button md-ink-ripple", type="button", ng-transclude="", … ]
​
autofocus: false
​
baseURI: "https://web-beta.threema.ch/#!/messenger/conversation/contact/*SUPPORT"
​
childElementCount: 1
​
childNodes: NodeList(3) [ #text, md-icon.ng-scope.material-icons, #text ]
​
children: HTMLCollection [ md-icon.ng-scope.material-icons ]
​
classList: DOMTokenList(3) [ "md-icon-button", "md-button", "md-ink-ripple" ]
​
className: "md-icon-button md-button md-ink-ripple"
​
clientHeight: 0
​
clientLeft: 0
​
clientTop: 0
​
clientWidth: 0
​
contentEditable: "inherit"
​
contextMenu: null
​
dataset: DOMStringMap(0)
​
dir: ""
​
disabled: false
​
draggable: false
​
firstChild: #text "
            "
​
firstElementChild: <md-icon class="ng-scope material-icons" role="img" aria-hidden="true">
​
form: null
​
formAction: "https://web-beta.threema.ch/#!/messenger/conversation/contact/*SUPPORT"
​
formEnctype: ""
​
formMethod: ""
​
formNoValidate: false
​
formTarget: ""
​
hidden: false
​
id: ""
​
innerHTML: "\n            <md-icon class=\"ng-scope material-icons\" role=\"img\" aria-hidden=\"true\"><img ng-src=\"img/ic_unpin.svg\" src=\"img/ic_unpin.svg\"></md-icon>\n        "
​
innerText: "\n            \n        "
​
isConnected: false
​
isContentEditable: false
​
labels: NodeList []
​
lang: ""
​
lastChild: #text "
        "
​
lastElementChild: <md-icon class="ng-scope material-icons" role="img" aria-hidden="true">
​
localName: "button"
​
name: ""
​
namespaceURI: "http://www.w3.org/1999/xhtml"
​
nextElementSibling: null
​
nextSibling: #text "
    "
​
ng339: undefined
​
nodeName: "BUTTON"
​
nodeType: 1
​
nodeValue: null
​
offsetHeight: 0
​
offsetLeft: 0
​
offsetParent: null
​
offsetTop: 0
​
offsetWidth: 0
​
onabort: null
​
onanimationcancel: null
​
onanimationend: null
​
onanimationiteration: null
​
onanimationstart: null
​
onauxclick: null
​
onblur: null
​
oncanplay: null
​
oncanplaythrough: null
​
onchange: null
​
onclick: null
​
onclose: null
​
oncontextmenu: null
​
oncopy: null
​
oncut: null
​
ondblclick: null
​
ondrag: null
​
ondragend: null
​
ondragenter: null
​
ondragexit: null
​
ondragleave: null
​
ondragover: null
​
ondragstart: null
​
ondrop: null
​
ondurationchange: null
​
onemptied: null
​
onended: null
​
onerror: null
​
onfocus: null
​
onfullscreenchange: null
​
onfullscreenerror: null
​
ongotpointercapture: null
​
oninput: null
​
oninvalid: null
​
onkeydown: null
​
onkeypress: null
​
onkeyup: null
​
onload: null
​
onloadeddata: null
​
onloadedmetadata: null
​
onloadend: null
​
onloadstart: null
​
onlostpointercapture: null
​
onmousedown: null
​
onmouseenter: null
​
onmouseleave: null
​
onmousemove: null
​
onmouseout: null
​
onmouseover: null
​
onmouseup: null
​
onmozfullscreenchange: null
​
onmozfullscreenerror: null
​
onpaste: null
​
onpause: null
​
onplay: null
​
onplaying: null
​
onpointercancel: null
​
onpointerdown: null
​
onpointerenter: null
​
onpointerleave: null
​
onpointermove: null
​
onpointerout: null
​
onpointerover: null
​
onpointerup: null
​
onprogress: null
​
onratechange: null
​
onreset: null
​
onresize: null
​
onscroll: null
​
onseeked: null
​
onseeking: null
​
onselect: null
​
onselectstart: null
​
onshow: null
​
onstalled: null
​
onsubmit: null
​
onsuspend: null
​
ontimeupdate: null
​
ontoggle: null
​
ontransitioncancel: null
​
ontransitionend: null
​
ontransitionrun: null
​
ontransitionstart: null
​
onvolumechange: null
​
onwaiting: null
​
onwebkitanimationend: null
​
onwebkitanimationiteration: null
​
onwebkitanimationstart: null
​
onwebkittransitionend: null
​
onwheel: null
​
outerHTML: "<button class=\"md-icon-button md-button md-ink-ripple\" type=\"button\" ng-transclude=\"\" translate-attr=\"{'aria-label': $ctrl.getLabel(), 'title': $ctrl.getLabel()}\" role=\"button\" tabindex=\"0\" ng-click=\"$ctrl.action()\" aria-pressed=\"false\" aria-label=\"Unterhaltung ist nicht angepinnt. Klicken, um sie anzupinnen.\" title=\"Unterhaltung ist nicht angepinnt. Klicken, um sie anzupinnen.\">\n            <md-icon class=\"ng-scope material-icons\" role=\"img\" aria-hidden=\"true\"><img ng-src=\"img/ic_unpin.svg\" src=\"img/ic_unpin.svg\"></md-icon>\n        </button>"
​
ownerDocument: HTMLDocument https://web-beta.threema.ch/#!/messenger/conversation/contact/*SUPPORT
​
parentElement: <toggle-button class="ng-scope ng-isolate-scope" flag="ctrl.conversation.isStarred" ng-if="ctrl.conversation" on-enable="ctrl.pinConversation()" on-disable="ctrl.unpinConversation()" label-enabled="messenger.PINNED_CONVERSATION" label-disabled="messenger.UNPINNED_CONVERSATION" icon-enabled="img/ic_pin.svg" icon-disabled="img/ic_unpin.svg">
​
parentNode: <toggle-button class="ng-scope ng-isolate-scope" flag="ctrl.conversation.isStarred" ng-if="ctrl.conversation" on-enable="ctrl.pinConversation()" on-disable="ctrl.unpinConversation()" label-enabled="messenger.PINNED_CONVERSATION" label-disabled="messenger.UNPINNED_CONVERSATION" icon-enabled="img/ic_pin.svg" icon-disabled="img/ic_unpin.svg">
​
prefix: null
​
previousElementSibling: null
​
previousSibling: #text "
        "
​
scrollHeight: 0
​
scrollLeft: 0
​
scrollLeftMax: 0
​
scrollTop: 0
​
scrollTopMax: 0
​
scrollWidth: 0
​
shadowRoot: null
​
slot: ""
​
spellcheck: false
​
style: CSS2Properties(0)
​
tabIndex: 0
​
tagName: "BUTTON"
​
textContent: "\n            \n        "
​
title: "Unterhaltung ist nicht angepinnt. Klicken, um sie anzupinnen."
​
type: "button"
​
validationMessage: ""
​
validity: ValidityState { valueMissing: false, typeMismatch: false, patternMismatch: false, … }
​
value: ""
​
willValidate: false
​
<prototype>: HTMLButtonElementPrototype { checkValidity: checkValidity(), reportValidity: reportValidity(), setCustomValidity: setCustomValidity(), … }
```
</details>

<hr>

```
Ein iframe, der sowohl "allow-scripts" als auch "allow-same-origin" für sein "sandbox"-Attribut gesetzt hat, kann seine Sandbox entfernen.
```

<hr>

```
onmozfullscreenchange sollte nicht mehr verwendet werden.
onmozfullscreenerror sollte nicht mehr verwendet werden.
```

## Your Environment

* Threema Web version: Version 2.2.0-beta.3 Glärnisch
* Threema App version: 4.0-beta2 (Android)
* Browser name and version: Firefox 67.0.4

**Comment by dbrgn** [comment]
> ARIA: Attribute " aria-label ", required for accessibility, is missing on node: 

We do set aria-label, albeit dynamically. This is a false positive.

> onmozfullscreenchange sollte nicht mehr verwendet werden.
> onmozfullscreenerror sollte nicht mehr verwendet werden.

Those don't come from our own source code.

> Ein iframe, der sowohl "allow-scripts" als auch "allow-same-origin" für sein "sandbox"-Attribut gesetzt hat, kann seine Sandbox entfernen.

We don't use iframes. Context?
